sci-biology/ABI_connectivity_data/ABI_connectivity_data-0.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs
+
+DESCRIPTION="Collection of connectivity data form the Allen Mouse Brain data portal"
```

Comment:
merely a style comment, but I think “Collection” here is unnecessary. “Connectivity data form the Allen Mouse Brain portal” provides the same level of information and is more concise.